src/components/NcAppSettingsDialog/NcAppSettingsDialogVersion.vue

```diff
@@ -0,0 +1,45 @@
+<!--
+  - SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+
+<script lang="ts">
+import { loadState } from '@nextcloud/initial-state'
+import { APP_NAME } from '../../utils/appName.ts'
+import { once } from '../../utils/utils.ts'
+
+const getLocalizedAppName = once(() => {
+	const activeApp = loadState<string>('core', 'active-app', APP_NAME)
+	return loadState<{ id: string, name: string }[]>('core', 'apps', []).find(({ id }) => id === activeApp)?.name ?? APP_NAME
+})
+</script>
```

Comment:
Can it be completely moved in utils? IMO it makes understanding of `once` usage complicated, if it's a part of component.

Reply:
Pushed fixup